src/Sylius/Bundle/ApiBundle/DependencyInjection/SyliusApiExtension.php

```diff
@@ -24,8 +24,8 @@
 class SyliusApiExtension extends AbstractResourceExtension implements PrependExtensionInterface
 {
     protected $configFiles = array(
```

Comment:
We can use patterns to avoid loading such files, but that's only idea for simplification.